feature/gnmi/ate_tests/telemetry_port_speed_test/telemetry_port_speed_test.go

```diff
@@ -185,6 +185,15 @@ func (tc *testCase) configureDUT(t *testing.T) {
 		tc.setupAggregateAtomically(t)
 	}
 
+	for _, port := range tc.dutPorts {
+		iName := port.Name()
+		i := &oc.Interface{Name: ygot.String(iName)}
+		tc.configMemberDUT(i, port)
+		iPath := d.Interface(iName)
+		fptest.LogQuery(t, port.String(), iPath.Config(), i)
+		gnmi.Replace(t, tc.dut, iPath.Config(), i)
+	}
+
```

Comment:
Did you try with this deviation instead?
[AggregateAtomicUpdate](https://source.corp.google.com/piper///depot/google3/third_party/openconfig/featureprofiles/internal/deviations/deviations.go;bpv=1;bpt=1;l=90?q=AggregateAtomicUpdate&gsn=AggregateAtomicUpdate&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fgoogle3%3Flang%3Dgo%3Fpath%3Dthird_party%2Fopenconfig%2Ffeatureprofiles%2Finternal%2Fdeviations%2Fdeviations%23var%2520AggregateAtomicUpdate) = [flag](http://godoc/pkg/flag).[Bool](https://source.corp.google.com/piper///depot/google3/third_party/go/gc/src/flag/flag.go;rcl=504995561;l=736)("deviation_aggregate_atomic_update", false,
		"Device requires that aggregate Port-Channel and its members be defined in a single gNMI Update transaction at /interfaces; otherwise lag-type will be dropped, and no member can be added to the aggregate.  Full OpenConfig compliant devices should pass both with and without this deviation.")